build/images/audit-controller/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.14
```

Comment:
let's not use alpine for the base image
let's use distroless / scratch whenever possible and ubuntu:20.04 otherwise